brainstormforce/ultimate-addons-for-gutenberg
# Fix: Collapsible menu not working on front-end in TOC block

### Description
Collapsible menu not working on front-end in TOC block

### Screenshots


### Types of changes
non-breaking change 

### Test case 1
- Switch to master branch
- Drag n drop TOC block and check front-end 
- Switch to fix-toc-issues branch and refresh the front-end 
- Check TOC working properly or not.

### Test Case 2
- Switch to fix-toc-issues branch.
- Drag and drop TOC block and check front-end.

### Checklist:
